pkg/auth/handlers.go

```diff
@@ -252,15 +251,18 @@ func GetMetadataEndpointRedirectHandler(ctx context.Context, authCtx interfaces.
 	}
 }
 
-// These are here for CORS handling. Actual serving of the OPTIONS request will be done by the gorilla/handlers package
-type CorsHandlerDecorator func(http.Handler) http.Handler
+func GetLogoutEndpointHandler(ctx context.Context, authCtx interfaces.AuthenticationContext) http.HandlerFunc {
+	return func(writer http.ResponseWriter, request *http.Request) {
+		logger.Debugf(ctx, "Deleting auth cookies")
```

Comment:
Just out of curiosity, are we in the habit of leaving in debug statements? They don't blog our staging/production logs right?

Reply:
I am yeah, it helps with debugging.  The logging level is controlled through a runtime config so we can play with it on the fly.